src/lib.rs

```diff
@@ -695,7 +670,7 @@ fn optimize_png(
         }
     }
 
-    eprintln!(
+    error!(
```

Comment:
@shssoichiro By the way, I'm not sure whether it's worth keeping this log message if it's duplicate in an actual error too (and will be printed e.g. by CLI app twice). Thoughts?

Reply:
I'm not entirely sure. But _theoretically_ this should never show up, and if it does, we'll definitely want the user to be aware of it. So I think it's fine to leave it.